internal/internal_task_pollers.go

```diff
@@ -326,10 +430,9 @@ func (wtp *workflowTaskPoller) RespondTaskCompleted(completedRequest interface{}
 
 	if err != nil {
 		wtp.metricsScope.Counter(metrics.DecisionResponseFailedCounter).Inc(1)
```

Comment:
May be move into RespondTaskCompletedWithMetrics as well?

Reply:
will update